tests/CMakeLists.txt

```diff
@@ -141,6 +141,11 @@ endfunction()
 # create a target to group all tests together into one test executable
 add_executable(test_all)
 
+# create a target to encapsulate all test categories, excluding test_all. This
+# is intended for conformance testing to avoid redundant testing and excessive
+# resource use from building and running test_all.
+add_custom_target(test_conformance)
```

Comment:
AFAIK the conformance script takes the build command in `-c` which is required. If others agree with this, I can go make it optional and have it default to just build this new target.

Reply:
> My only concern is that it may be difficult for users to discover that the `test_conformance` target exists.

Should we update the README to explicitly mention those two special targets (I mean `test_all` as a second one).

Also: should we consider removing `test_all` from the default `all` target? [CMake docs on that](https://cmake.org/cmake/help/v3.20/prop_tgt/EXCLUDE_FROM_ALL.html)